src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,45 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.6.0
+13 March, 2023
+
+### Deprecation Notice
+- Support for Kubernetes [1.21](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.21.md) has been deprecated and will be removed next release.
+
+### Features
```

Comment:
That is part of updating the CLI used in the backend. Do we want to mention explicitly also that? Wouldn't be enough with the link to the release notes of `2.13.0`?

Reply:
No, let's mention the feature explicitly.  Are there any other major CLI features? If so, we should also mention them here since they enable new scenarios on Okteto Self-Hosted.

Reply:
ok. The other big feature is the remote deploy and destroy but not sure it worths mentioning here as everything deployed through the UI is already done remotely, so from a user perspective might be confusing to mentioning here. The real difference happens when you run `okteto deploy` locally

Reply:
I still think that we should mention it. But it's not a blocker. I'll ask Pablo and Matt to follow up on this.

Reply:
👍 is there anything else blocking the PR? If not, we could merge it and finish the release process

Reply:
No, merge and release.

Please don't forget to make 1.6 the default version once 1.6 is available 👍🏽